src/core/Akka/Dispatch/Dispatchers.cs

```diff
@@ -209,6 +209,7 @@ public MessageDispatcher Lookup(string dispatcherName)
         public MessageDispatcher FromConfig(string path)
         {
             //TODO: this should not exist, it is only here because we dont serialize dispathcer when doing remote deploy..
+            //TODO: @Aaron, we do now, should this be removed?
```

Comment:
@rogeralsing yes, we should remove this and then re-run the `RemoteDeploy` tests in Akka.Remote. Everything should go through.